test/saithrift/src/saiserver.cpp

```diff
@@ -47,7 +49,80 @@ void on_switch_state_change(_In_ sai_object_id_t switch_id,
 void on_fdb_event(_In_ uint32_t count,
                   _In_ sai_fdb_event_notification_data_t *data)
 {
-}
+    sai_fdb_event_t event_type;
+    sai_fdb_entry_t fdb_entry;
+    uint32_t attr_count;
+    sai_attribute_t *attr;
+    sai_object_id_t bv_id;	
+    sai_object_id_t bport_id;
+	
+    attr = data->attr;
+    event_type = data->event_type;
+    fdb_entry = data->fdb_entry;
+    bv_id = fdb_entry.bv_id;
+    attr_count = data ->attr_count;
+    
+    for (uint32_t i = 0; i < attr_count; i++)
+    {
+        if(attr[i].id == SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID)
+            bport_id = attr[i].value.oid;
+    }
+           
+    sai_fdb_entry_t fdb_m;
+    sai_object_id_t b_id;
+      
+    switch(event_type)
+    {   
+        case SAI_FDB_EVENT_LEARNED
+            gFdbMap.insert(std::pair<std::sai_fdb_entry_t,std::sai_fdb_event_t>(fdb_entry,bport_id));  
```

Comment:
Type doesn't match `sai_object_id_t `instead of `sai_fdb_event_t`

Reply:
updated as commented